src/webgl/interaction.js

```diff
@@ -42,7 +45,11 @@ import * as constants from '../core/constants';
  * }
  * function draw() {
  *   background(200);
+ *
+ *   // If you write here like orbitControl(1, 1, 1, {free: true})
+ *   // instead of this, the behavior will change.
```

Comment:
Maybe instead of saying "the behavior will change" we can say something similar to line 33?

Reply:
I didn't explain that part enough...
How about this?
```js
/**
 *   // If you execute the line commented out instead of next line, the direction of rotation
 *   // will be the direction the mouse or touch pointer moves, not around the X or Y axis.
 *   orbitControl();
 *   // orbitControl(1, 1, 1, {"optionName": true});
 */
```

Reply:
sounds good!